diff -up qtmultimedia-everywhere-src-6.6.0-beta3/examples/multimedia/screencapture/CMakeLists.txt.1~ qtmultimedia-everywhere-src-6.6.0-beta3/examples/multimedia/screencapture/CMakeLists.txt
--- qtmultimedia-everywhere-src-6.6.0-beta3/examples/multimedia/screencapture/CMakeLists.txt.1~	2023-08-16 22:35:08.000000000 +0200
+++ qtmultimedia-everywhere-src-6.6.0-beta3/examples/multimedia/screencapture/CMakeLists.txt	2023-08-22 21:26:07.544094461 +0200
@@ -11,8 +11,8 @@ set(CMAKE_AUTORCC ON)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Widgets)
-find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS MultimediaWidgets Widgets)
+find_package(QT NAMES Qt6 REQUIRED COMPONENTS Widgets)
+find_package(Qt6 REQUIRED COMPONENTS MultimediaWidgets Widgets)
 
 set(PROJECT_SOURCES
         screencapture.pro
@@ -31,11 +31,11 @@ qt_add_executable(screencapture
     )
 
 target_link_libraries(screencapture PRIVATE
-    Qt${QT_VERSION_MAJOR}::Core
-    Qt${QT_VERSION_MAJOR}::Gui
-    Qt${QT_VERSION_MAJOR}::Widgets
-    Qt${QT_VERSION_MAJOR}::Multimedia
-    Qt${QT_VERSION_MAJOR}::MultimediaWidgets)
+    Qt6::Core
+    Qt6::Gui
+    Qt6::Widgets
+    Qt6::Multimedia
+    Qt6::MultimediaWidgets)
 
 set_target_properties(screencapture PROPERTIES
     MACOSX_BUNDLE_INFO_PLIST ${CMAKE_CURRENT_SOURCE_DIR}/Info.plist.in
